addons/website_slides/static/src/xml/slide_quiz.xml

```diff
@@ -3,7 +3,7 @@
     <t t-name="slide.slide.quiz">
         <div class="o_wslides_fs_quiz_container o_wslides_wrap h-100 w-100 overflow-auto pb-5">
             <div class="container">
-                <div t-if="widget.quiz.description_safe" t-esc="widget.quiz.description_safe" class="mt-3" />
```

Comment:
Hello @thsh-odoo 

Do we know what induced this change?
Would be nice to have a reference to that commit in your commit message, otherwise it's a bit hard to know why you target master (16.5) at this was introduced in 16.0.

Otherwise LGTM.

Thanks & Cheers!